yt/data_objects/static_output.py

```diff
@@ -1060,7 +1059,7 @@ def _assign_unit_system(self, unit_system):
         self.unit_registry.unit_system = self.unit_system
 
     def _create_unit_registry(self, unit_system):
-        import yt.units.dimensions as dimensions
+        from yt.units import dimensions as dimensions
```

Comment:
Is there a specific reason for this import to happen here instead of the beginning of the file?

Reply:
I don't believe so. Fixing.